webplugin/js/app/kommunicate-ui.js

```diff
@@ -515,6 +515,9 @@ handleAttachmentIconVisibility : function(enableAttachment, msg, groupReloaded)
             popupTemplateKey === KommunicateConstants.CHAT_POPUP_TEMPLATE.HORIZONTAL ? kommunicateIframe.classList.add('chat-popup-widget-horizontal') : kommunicateIframe.classList.add('chat-popup-widget-vertical');
             kommunicateCommons.modifyClassList( {id : ["launcher-svg-container"]}, "km-animate", "");
             kommunicateCommons.modifyClassList( {id : ["chat-popup-widget-container"]}, "km-animate", "n-vis");
+            var moveWidgetToLeft = kommunicate && kommunicate._globals && kommunicate._globals.widgetSettings && kommunicate._globals.widgetSettings.hasOwnProperty('moveWidgetToLeft') && kommunicate._globals.widgetSettings.moveWidgetToLeft;
```

Comment:
can this be changed to something else, move is like when something is initially positioned somewhere and then moved dynamically. this is more like a position from the beginning.

Reply:
in future if add more positions, it will be difficult to maintain multiple properties, something like below can be used

position: 'bottom-right' (default)
position: 'bottom-left'
position: 'top-right' and so on...